### PR TITLE
Reduce log spam of load shed points if not configured. 

### PR DESCRIPTION
**Pull request by KBaichoo**
Commit Message: Reduce log spam of load shed points if not configured. 
Additional Description: LOG_ONCE_IF in the various callsites so each point gets logged about at most once.
Risk Level: low
Testing: unit test
Docs Changes: na
Release Notes: na
Platform Specific Features: na
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: test/common/common/log_macros_test.cc [hunk 1]
```diff
@@ -282,7 +319,7 @@ class FormatTest : public testing::Test {
 public:
   static void logMessageEscapeSequences() {
     ENVOY_LOG_MISC(info, "line 1 \n line 2 \t tab \\r test");
-  }
+  } // namespace Envoy
```
Comment (adisuissa) [inline test/common/common/log_macros_test.cc:322]: Is the comment correct?

**Review by adisuissa — COMMENTED** [review]
Thanks!
LGTM, with a small nit.

File: test/common/common/log_macros_test.cc (hunk 1 above)
Comment (KBaichoo) [reply test/common/common/log_macros_test.cc]: nope, not sure how that happened. removed.

**Review by adisuissa — APPROVED** [review]
LGTM, thanks!

**Pull request merged**